spec/factories/activities.rb

```diff
@@ -11,6 +11,7 @@
     always_on { false }
     retired { false }
     self_verification_info { nil }
+    duration_in_hours { 2.5 }
```

Comment:
I wonder if we should have this default to nil, as that currently the default behaviour we have on seeds, then maybe have a trait :with_duration?

Reply:
Just having a think about this. I need to test cs_accelerator cases with different durations (less than a day, more than one day).
Would it be cleaner to define the duration in hours when creating the activity for the individual test, rather than a trait? Otherwise I'd have to create multiple traits. 
Eg. `activity = create(:activity, :with_duration, credit: nil, programmes: [cs_accelerator_programme], duration_in_hours: 10)`

Reply:
...and have the default to nil as suggested.